src/kernel/core/file.cc

```diff
@@ -22,13 +22,14 @@ u32	File::inode_system=0;	/* numero d'inode de depart */
 
 /* constructeur */
 File::File(char* n,u8 t){
-	name=(char*)kmalloc(strlen(n)+1);
-	memset(name,0,strlen(n));
-	memcpy(name,n,strlen(n));
-	name[strlen(n)]=0;
+	size_t length = strlen(n);
+	name=(char*)kmalloc(length+1);
+	memset(name,0,length);
```

Comment:
Master wrote it so I do not know why it is important or not. I only saw unnecessary function calls and fixed it.